src/themes/default/lv_theme_default.c

```diff
@@ -263,6 +264,7 @@ static void style_init(my_theme_t * theme)
     lv_style_set_text_font(&theme->styles.scr, theme->base.font_normal);
     lv_style_set_pad_row(&theme->styles.scr, PAD_SMALL);
     lv_style_set_pad_column(&theme->styles.scr, PAD_SMALL);
+    lv_style_set_rotary_sensitivity(&theme->styles.scr, theme->disp_dpi / 4 * 256);
```

Comment:
`4 * 256` sounds like a magic number.
I assume you have tested with SDL, so I wonder if it's a good value for real devices, too.